[WIP] Documentation fixes and enhancements (#584)
pytorch/serve: docs/code_coverage.md

## To check branch stability run the sanity suite as follows

> Ensure that node dependency are already installed in your system. Refer [Install Node dependency](#install-markdown-link-checker-dependencies)  

Reviewer: NIT:
```suggestion
Ensure that node dependency is already installed on your system. Refer [Install Node dependency](#install-markdown-link-checker-dependencies)  
```
